agent/util-scripts/pbench-move-results

```diff
@@ -172,120 +186,84 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
         /bin/cp pbench.log $pbench_run_name/
     fi
 
+    # if -u was specified, store the specified user in metadata.log
+    if [ ! -z $user ] ;then
+	mdlog=${pbench_run_name}/metadata.log
+	echo $user | pbench-add-metalog-option ${mdlog} run user
+    fi
+
+    # if -p was specified, store the specified prefix in metadata.log
+    if [ ! -z $prefix ] ;then
+	mdlog=${pbench_run_name}/metadata.log
+	echo $prefix | pbench-add-metalog-option ${mdlog} run prefix
+    fi
+
     results_size=`du -sm $pbench_run_name | awk '{print $1}'`
     debug_log "preparing to copy $results_size MB of data from $pbench_run/$pbench_run_name"
 
-    tarball="$pbench_run_name.tar.xz"
+    # Create a temp directory $tmp/$controller to contain the tarball
+    # and the md5 file (as ${tb}.tar.xz.md5.check). Copy the directory
+    # with scp -r $tmp/$controller $remote: that will create the
+    # $controller subdirectory on the remote (if necessary) OR fail.
+
+    # If it does not fail, then check the MD5 sum and rename the foo.tar.xz.md5.check file
+    # to foo.tar.xz.md5. That's the signal that the agent has finished with this tarball.
+
+    tarball="$tmp/$controller/$pbench_run_name.tar.xz"
```

Comment:
This URL will be wrong because of above forward slash in tmp.

Reply:
@tenstormavi, you mean the there will be a 2 slash characters in a row in the URL?

Where will `$tarball` get used as a URL?